collector/remote_fx.go

```diff
@@ -0,0 +1,472 @@
+			[]string{"session"},
+			nil,
+		),
+		FECRate: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_fec_rate"),
+			"Forward Error Correction (FEC) percentage",
+			[]string{"session"},
+			nil,
+		),
+		LossRate: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_loss_rate"),
+			"Loss percentage",
+			[]string{"session"},
+			nil,
+		),
+		RetransmissionRate: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_retransmission_rate"),
+			"Percentage of packets that have been retransmitted",
+			[]string{"session"},
+			nil,
+		),
+		TCPReceivedRate: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_tcp_received_rate"),
+			"Rate in bits per second (bps) at which data is received over TCP.",
+			[]string{"session"},
+			nil,
+		),
+		TCPSentRate: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_tcp_sent_rate"),
+			"Rate in bits per second (bps) at which data is sent over TCP.",
+			[]string{"session"},
+			nil,
+		),
+		TotalReceivedRate: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_total_received_rate"),
+			"Rate in bits per second (bps) at which data is received.",
+			[]string{"session"},
+			nil,
+		),
+		TotalSentRate: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_total_sent_rate"),
+			"Rate in bits per second (bps) at which data is sent.",
+			[]string{"session"},
+			nil,
+		),
+		TotalReceivedBytes: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_total_received_bytes"),
+			"(TotalReceivedBytes)",
+			[]string{"session"},
+			nil,
+		),
+		TotalSentBytes: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_total_sent_bytes"),
+			"(TotalSentBytes)",
+			[]string{"session"},
+			nil,
+		),
+		UDPPacketsReceivedPersec: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_udp_packets_received_persec"),
+			"Rate in packets per second at which packets are received over UDP.",
+			[]string{"session"},
+			nil,
+		),
+		UDPPacketsSentPersec: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_udp_packets_sent_persec"),
+			"Rate in packets per second at which packets are sent over UDP.",
+			[]string{"session"},
+			nil,
+		),
+		UDPReceivedRate: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_udp_received_rate"),
+			"Rate in bits per second (bps) at which data is received over UDP.",
+			[]string{"session"},
+			nil,
+		),
+		UDPSentRate: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_udp_sent_rate"),
+			"Rate in bits per second (bps) at which data is sent over UDP.",
+			[]string{"session"},
+			nil,
+		),
+
+		//gfx
+		AverageEncodingTime: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "gfx_average_encoding_time"),
+			"Average frame encoding time.",
```

Comment:
What unit does this metric represent? Seconds? Milliseconds?

Reply:
Milliseconds updating help.